include/mixer.h

```diff
@@ -480,14 +484,15 @@ void MIXER_SetReverbPreset(const ReverbPreset new_preset);
 ChorusPreset MIXER_GetChorusPreset();
 void MIXER_SetChorusPreset(const ChorusPreset new_preset);
 
-// Generic callback used for audio devices which generate audio on the main thread
-// These devices produce audio on the main thread and consume on the mixer thread
-// This callback is the consumer part
+// Generic callback used for audio devices which generate audio on the main
```

Comment:
The beautifier went too far here. I suggest to manually adjust line breaks, or introduce full stops at the end of sentence.